### PR TITLE
feat: Update readme with Feature Key Tracking feature

### PR DESCRIPTION
**Pull request by yavorona**
This pr updates README.md with feature key tracking feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Optimizely VSCode Extension
 
-The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript typescript and react.  If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
+The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript, typescript and react. If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
 
-This extension is compatiable with 1.34 and higher of vscode.
+This extension also evaluates the use of Optimizely's public APIs (e.g. getFeatureVariableJSON(feature_key, userId, attributes)) in active .js and .ts files, and highlights Feature Keys that do not exist in datafile.
```
Comment (fscelliott) [inline README.md:5]: I'd reword as:
This extension also evaluates if parameter values are valid for Optimizely's public SDK methods (for example, getFeatureVariable(feature_key, userID, attributes), or isFeatureEnabled(feature_key)) for Javascript and Typescript. If a given parameter is invalid (i.e., doesn't exist in the datafile), this extension highlights it and displays a warning on hover.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Optimizely VSCode Extension
 
-The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript typescript and react.  If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
+The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript, typescript and react. If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
```
Comment (fscelliott) [inline README.md:3]: The Optimizely VSCode extension autocompletes:
- feature keys
- experiment keys
- feature variables 
in Javascript, Typescript, and React.

QUESTION: it's not clear if In .js, .ts, and react, you can also autocomplete attribute and event keys without keyboard shortcuts? if you can, please add them to the list and modify the second sentence:
In other languages, you can insert the preceding list of elements using the following keyboard shortcuts.

File: README.md (hunk 1 above)
Comment (yavorona) [reply README.md:5]: I think this is great! I would just specify what parameter values being evaluated, which are only feature keys at the moment:

This extension also evaluates if `feature keys` are valid for Optimizely's public SDK methods (for example, getFeatureVariable(feature_key, userID, attributes), or isFeatureEnabled(feature_key)) for Javascript and Typescript. If a given `feature_key` is invalid (i.e., doesn't exist in the datafile), this extension highlights it and displays a warning on hover.

File: README.md (hunk 2 above)
Comment (yavorona) [reply README.md:3]: I believe we can, so they should be added to the list. @thomaszurkan-optimizely WDYT?

Comment (thomaszurkan-optimizely) [reply README.md:3]: events and attributes don't auto-complete.

Comment (yavorona) [reply README.md]: I added back events tags since we added it in our last #14 pr

File: README.md [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+The Optimizely VSCode extension autocompletes the following parameters in Javascript, Typescript, and React:
+- feature keys
+- experiment keys
+- feature variables
+- event keys
 
-This extension is compatiable with 1.34 and higher of vscode.
+In other languages, you can insert the preceding list of elements using the following keyboard shortcuts.
+
+This extension also evaluates if provided feature keys are valid for Optimizely's public SDK methods (for example, getFeatureVariable(feature_key, userID, attributes), or isFeatureEnabled(feature_key, userID)) for Javascript and TypeScript. If a given feature_key is invalid (i.e., doesn't exist in the datafile), the extension highlights it and displays a warning on hover.
```
Comment (thomaszurkan-optimizely) [inline README.md:11]: Can we change this slightly to say something like: "(i.e. doesn't exist in the datafile for the sdk key used to configure)."

**Review by thomaszurkan-optimizely — CHANGES_REQUESTED** [review]
nit

**Review by thomaszurkan-optimizely — APPROVED** [review]
LGTM
